support-frontend/assets/components/personalDetails/personalDetails.tsx

```diff
@@ -87,6 +87,7 @@ export function PersonalDetails({
 					pattern={emailRegexPattern}
 					error={errors?.email?.[0]}
 					disabled={isSignedIn}
+					name="email"
```

Comment:
By adding the `name` fields this becomes available on the `FormData` [when passing the form element through to it's constructor](https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData#prepopulating_from_a_html_form_element).

Reply:
Presume so we can use for example 'document.getElementById("email")'